src/helm/benchmark/presentation/run_specs.conf

```diff
@@ -92,49 +92,49 @@ entries: [
   # Scenarios: MS Marco (Regular), MS MARCO (TREC)
 
   {description: "msmarco:model=full_functionality_text,data_augmentation=canonical,track=regular,valid_topk=30", priority: 2}
-  {description: "msmarco:model=full_functionality_text,data_augmentation=canonical,track=trec,valid_topk=30", priority: 1}
+  {description: "msmarco:model=full_functionality_text,data_augmentation=canonical,track=trec,valid_topk=30", priority: 2}
 
   ##### Summarization #####
   # Scenarios: XSUM, CNN/DM
 
-  {description: "summarization_cnndm:model=text,temperature=0.3,device=cpu", priority: 1}
-  {description: "summarization_xsum_sampled:model=text,temperature=0.3,device=cpu", priority: 1}
+  {description: "summarization_cnndm:model=text,temperature=0.3,device=cpu", priority: 2}
+  {description: "summarization_xsum_sampled:model=text,temperature=0.3,device=cpu", priority: 2}
 
 
   ##### Sentiment Analysis #####
   # Scenarios: IMDB
 
-  {description: "imdb:model=text,data_augmentation=canonical", priority: 1}
+  {description: "imdb:model=text,data_augmentation=canonical", priority: 2}
```

Comment:
Is IMDB causing problems?

Reply:
I would keep IMDB priority 1, and more generally keep all of the text classification in IMDB, CivilComments, and RAFT unless there is a reason not to (since I assume they should be cheap to run and not cause breaking).

Reply:
I moved this back to priority 1. imdb is relatively cheap. The original reasoning from another discussion that it was saturated.